README.md

```diff
@@ -5,3 +5,128 @@
+
+After you fork and clone this project, run `npm install` and `npm start`. What do you see? Can you identify what you see and which component it comes from?
+
+### A Note About Styles
+
+We have provided styles for you in `.css` files. Styling is not a major learning goal for this project, so please limit the amount of time you spend on styling.
+
+To ease the use of our styles, the project conforms to a naming methodology/naming convention standard known as [BEM, or Block Element Modifier](http://getbem.com/). Essentially, you can count on the provided styles to conform to the following naming rules:
+- There aren't any, or rarely any element/tag selectors or ID selectors, and all styles selectors are on classes. That means there are are a lot more classes to set, but it relies less on assuming a specific HTML structure before understanding how to use it
+- Classes are named in [block__element--modifier](http://getbem.com/naming/) format:
+  - The first section, `block`, will describe the semantic block that the style applies to
+  - The second section, `element` (after two underscores), will describe the specific element within that block that the style applies to
+  - The third section `modifier` (after two hyphens), will describe any sort of modifier (ie `enabled`, `success`, `green`) for that block and element that the style applies to
+
+Here's an example:
+```css
+.new-card-form__header {
+  text-align: center;
+  ...
+}
+```
+
+In this case, there is some "block" (section, idea, maybe component) named `new-card-form`, and this style is for the "element" that represents the header by name of `header`. This style makes the header in the new card form `text-align: center;`. Note that this does not dictate what kind of HTML tag this is, or what its parent or children are.
+
+Again, please limit the time you spend on styling, and reach out often and frequently.
+
+## Requirements
+
+### Wave 0: Establish the Relationships
+
+1. Make a diagram with all of the components, illustrating which components are nested within each other, and making it clear which components are siblings to each other.
+1. For each component, list out what data needs to be referenced within it, and what data type each piece of data is. Particularly, think about:
+    - If the `PlayerSubmissionForm` component handles a player's submission of poetry, what should that component do with that data?
+    - How should the `FinalPoem` component get all of the parts of the poem?
+    - What kind of input elements should `PlayerSubmissionForm` use? What does this mean about the `PoemInput` component?
+    - What should the `Game` component represent and be responsible for?
+1. For each piece of data in each component, determine if it should be part of that component's `state` or passed in as a `prop`.
+
+The waves are organized in this order:
```

Comment:
Maybe describe the waves before going into wave 0.

Reply:
I actually looked at this and I think I want to keep this section here. Previewing the waves here talks about the components in detail, and I think I'd want to design this activity such that they don't preview the waves in detail until they've diagrammed the components they've seen themselves as detailed in wave 0, otherwise it will overwhelm them